cdc/sink/dmlsink/mq/dispatcher/partition/index_value_test.go

```diff
@@ -183,13 +183,14 @@ func TestIndexValueDispatcherWithIndexName(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, int32(2), index)
 
+	idx := index
```

Comment:
Just a confirm: can the test case cover the case that not using the original column name can not pass the test?

Reply:
If the column names are not the original column names, the partition should match the original column names.